frontend/src/main/utils/systemInfo.js

```diff
@@ -6,6 +6,7 @@ export function useSystemInfo() {
   return useQuery("systemInfo", async () => {
     try {
       const response = await axios.get("/api/systemInfo");
+      //document.write(response?.springH2ConsoleEnabled)
```

Comment:
Please consider removing this commented out line of code.

We try to avoid committing commented out code to the main branch unless there is a compelling reason to do so.